tests/ludwig/schema/test_schema_fields.py

```diff
@@ -0,0 +1,27 @@
+from ludwig.constants import ALLOF, ENCODER, ENUM, INPUT_FEATURES, ITEMS, PROPERTIES, THEN
```

Comment:
If we keep these tests, add them tests to `tests/ludwig/config_validation/`

Reply:
Done!